### PR TITLE
[do not merge] bump newtonsoft json version

### PR DESCRIPTION
**Pull request by maddieclayton**
## Description

---

This checklist is used to make sure that common guidelines for a pull request are followed.
- [ ] Please add REST spec PR link to the SDK PR
- [ ] **I have read the [contribution guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md).**
- [ ] **The pull request does not introduce [breaking changes](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/Documentation/breaking-changes.md).**

### [General Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#general-guidelines)
- [ ] Title of the pull request is clear and informative.
- [ ] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/dev/documentation/cleaning-up-commits.md).

### [Testing Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#testing-guidelines)
- [ ] Pull request includes test coverage for the included changes.

### [SDK Generation Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#sdk-generation-guidelines)
- [ ] If an SDK is being regenerated based on a new swagger spec, a link to the pull request containing these swagger spec changes has been included above.
- [ ] The generate.cmd file for the SDK has been updated with the version of AutoRest, as well as the commitid of your swagger spec or link to the swagger spec, used to generate the code.
- [ ] The `*.csproj` and `AssemblyInfo.cs` files have been updated with the new version of the SDK.

### CODE REVIEW

File: src/SDKs/SignalR/Management.SignalR/Properties/AssemblyInfo.cs [hunk 1]
```diff
@@ -8,8 +8,8 @@
 [assembly: AssemblyTitle("Microsoft Azure SignalR Management Library")]
 [assembly: AssemblyDescription("Provides management functionality for Microsoft Azure SignalR Resources.")]
 
-[assembly: AssemblyVersion("0.10.0.0")]
-[assembly: AssemblyFileVersion("0.10.0.0")]
+[assembly: AssemblyVersion("0.10.1.0")]
```
Comment (shahabhijeet) [inline src/SDKs/SignalR/Management.SignalR/Properties/AssemblyInfo.cs]: @maddieclayton update this version to 0.11.0 (both assemblyversion as well assemblyfile version)

File: src/SdkCommon/TestFramework/Microsoft.Azure.Test.HttpRecorder/Properties/AssemblyInfo.cs [hunk 2]
```diff
@@ -11,12 +11,12 @@
...
 [assembly: AssemblyVersion("1.0.0.0")]
-[assembly: AssemblyFileVersion("1.13.0.0")]
+[assembly: AssemblyFileVersion("1.13.1.0")]
 [assembly: AssemblyTitle("Microsoft.Azure.Test.HttpRecorder")]
 [assembly: AssemblyDescription("")]
 [assembly: AssemblyConfiguration("")]
 [assembly: AssemblyCompany("")]
 [assembly: AssemblyProduct("Microsoft.Azure.Test.HttpRecorder")]
-[assembly: AssemblyCopyright("Copyright �")]
+[assembly: AssemblyCopyright("Copyright �")]
```
Comment (shahabhijeet) [inline src/SdkCommon/TestFramework/Microsoft.Azure.Test.HttpRecorder/Properties/AssemblyInfo.cs]: @maddieclayton can you bring back the copyright character

**Review by shahabhijeet — CHANGES_REQUESTED** [review]
Versioning Rules to follow:
1. If you see anything that has 0.x as the Asm and AsmFile version, simply bump minor version for both Asm and AsmFile version (e.g. 0.10.0 goes to 0.11.0)
2. If you see a non-zero major version in both Asm and AsmFile version, bump only revision number of the AsmFileVersion, do not make any updates to Asm version (e.g. old version 2.0.0 (asm version) and 2.13.0 (AsmFile Version), only bump AsmFile version to 2.13.1

Add the following text in ReleaseNotes every SDK who we are planning to re-release as below:
Taking dependency on 10.0.3 version of Newntonsoft nuget package.

File: src/SDKs/Search/Management/Management.Search/Microsoft.Azure.Management.Search.csproj [hunk 3]
```diff
@@ -8,9 +8,9 @@
 		<Description>Makes it easy to manage Azure Search services and API keys from a .NET application.</Description>
 		<AssemblyTitle>Microsoft Azure Search Management Library</AssemblyTitle>
 		<AssemblyName>Microsoft.Azure.Management.Search</AssemblyName>
-		<Version>2.0.0</Version>
+		<Version>2.0.1</Version>
 		<PackageTags>Microsoft Azure Search;Microsoft Azure Search Management;</PackageTags>
-		<PackageReleaseNotes>This is the Azure Search Management SDK for .NET, based on version 2015-08-19 of the Azure Search Management REST API. It includes support for programmatically provisioning and managing Search services and API keys. It supports all available SKUs, including the new S3 and S3 High-Density SKUs.</PackageReleaseNotes>
+		<PackageReleaseNotes>Taking dependency on 10.0.3 version of Newntonsoft nuget package.</PackageReleaseNotes>
```
Comment (brjohnstmsft) [inline src/SDKs/Search/Management/Management.Search/Microsoft.Azure.Management.Search.csproj]: Add new information if you have to, but please don't remove the existing release notes.

**Review by bitsforthought — CHANGES_REQUESTED** [review]

**Review by bitsforthought — APPROVED** [review]

**Pull request closed without merging**